src/components/AdminPortal/TagActions/TagActions.module.css

```
@@ -1,33 +1,98 @@
 }
 
-.removeFilterIcon {
-  composes: removeFilterIcon from '../../../style/app-fixed.module.css';
+.searchSection {
+  margin-top: var(--space-2);
+  margin-bottom: var(--space-1);
 }
 
-.inputField {
-  composes: inputField from '../../../style/app-fixed.module.css';
+.removeMemberChipButton:global(.btn) {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)